provisioning/environments/e2e_vulnerability_detector.yaml

```diff
@@ -35,8 +35,8 @@ agent5:
     manager: manager2
     architecture: arm64v8
 
-agent6:
-    roles: [agent]
-    os: macos_1400
-    manager: manager1
-    architecture: arm64v8
+# agent6:
+#     roles: [agent]
+#     os: macos_1400
+#     manager: manager1
+#     architecture: arm64v8
```

Comment:
This was introduced for testing, we should remove it.

Reply:
Done e970ae56dd9949ea910cb16cb813bbb28b07e698